packages/core/src/docdb/utils.ts

```diff
@@ -0,0 +1,149 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { localize } from '../shared/utilities/vsCodeUtils'
+
+/**
+ * Validates a cluster name for the CreateCluster API.
+ * @see https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/docdb/command/CreateDBClusterCommand/
+ * @returns undefined if the name passes validation. Otherwise, an error message is returned.
+ */
+export function validateClusterName(name: string): string | undefined {
```

Comment:
Does the service not return validation errors that we can simply pass along?

Reply:
It does, but we want to be proactive where possible especially as this is part of a multi-step wizard.